var/spack/repos/builtin/packages/autodock-gpu/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class AutodockGpu(MakefilePackage):
+    """AutoDock-GPU: AutoDock for GPUs and other accelerators.
+    OpenCL and Cuda accelerated version of AutoDock 4.2.6. It
+    leverages its embarrasingly parallelizable LGA by processing
+    ligand-receptor poses in parallel over multiple compute units.
+    """
+
+    homepage = "https://ccsb.scripps.edu/autodock"
+    git      = "https://github.com/ccsb-scripps/AutoDock-GPU.git"
+
+    maintainers = ['RemiLacroix-IDRIS']
+
+    version('develop', branch='develop')
+
+    variant('device', default='cuda', description='Acceletor runtime',
+            values=('cuda', 'oclgpu'), multi=False)
+    variant('overlap', default=False, description='Overlap CPU and GPU operations')
+
+    depends_on('cuda')
+
+    @property
+    def build_targets(self):
+        spec = self.spec
+        return [
+            'DEVICE={0}'.format(spec.variants['device'].value.upper()),
+            'GPU_INCLUDE_PATH={0}'.format(spec['cuda'].prefix.include),
+            'GPU_LIBRARY_PATH={0}'.format(spec['cuda'].prefix.lib64),
```

Comment:
Is it always guaranteed to be `lib64` on every system? It's probably better to use `spec['cuda'].libs.directories[0]`

Reply:
I wasn't completely sure myself, I think your suggestion is better.